content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -4,18 +4,23 @@ This document describes how we release Sourcegraph.
 
 ## Release policies
 
-> As of Feb 3, 2022, the role of Release Captain is now rotation-based within the [Cloud](../../cloud/index.md) org, [learn more](https://sourcegraph.slack.com/archives/C0EPTDE9L/p1643915838386829). If you need assistance with releases, please reach out to [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX). The [Delivery] team is more than happy to help you through the process.
+> As of [RFC 612](https://docs.google.com/document/d/1Gecnsk4mnmf_p9SO4ExICSTC_op-eL2I_kwutHvwSmU/edit) the ownership of the release process has changed.
+> The [Release Guild](./release_guild.md) is now the entity that owns the release process. Read more details about release
+> responsibilities in the [Releases](./index.md#Releases) section below.
 
 ### Releases
 
-**Sourcegraph releases are monthly.**
+**Sourcegraph minor releases are monthly.**
```

Comment:
Begs the question, when will we ever do a major release? And if so, who owns that?

Reply:
It's been years for a major release 😂

I think the proposed change in the title at least prevents from release guild automatically owning the major ones, and is technically correct (to me), which forces the leadership to think about it when we plan to make a major release.